### DISCUSSION TITLE
How to increase the frequency at which ApplicationSet changes are refreshed?

### DISCUSSION THREAD

**Discussion by gajus** [argoproj/argo-cd#15997] [Q&A]
What is the setting to increase the frequency at which ApplicationSet changes are synced with the repository? At the moment it appears to be 3 minutes.

**Comment by naruse666** [comment]
@gajus 
This will help the problem you are facing.
https://argo-cd.readthedocs.io/en/stable/faq/#how-often-does-argo-cd-check-for-changes-to-my-git-or-helm-repository

if you installed helm, you can set on `.config.cm.timeout.reconciliation`.
https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml

**Reply by gajus** [reply]
ApplicationSet does not respect that setting https://github.com/argoproj/argo-cd/issues/13047

**Reply by naruse666** [reply]
I didn't know that exist this issue.
if nobody answered your comment, could have better to ask on slack.

I'll take a look at the related source code when I have time.
